utils/swift_build_support/README.md

```diff
@@ -6,5 +6,5 @@ structures used by the Swift build script.
 You may run unit tests for `swift_build_support` from the command line:
 
 ```sh
-$ python utils/swift_build_support/run_tests.py
```

Comment:
While a reasonable fix, this change looks unrelated

Reply:
makes sense, I'll submit it as a separate PR